File: src/main/java/maple/expectation/global/error/exception/auth/DuplicateLikeException.java
package maple.expectation.global.error.exception.auth;

import maple.expectation.global.error.CommonErrorCode;
import maple.expectation.global.error.exception.base.ClientBaseException;
import maple.expectation.global.error.exception.marker.CircuitBreakerIgnoreMarker;

/**
 * 중복 좋아요 예외
 */
public class DuplicateLikeException extends ClientBaseException implements CircuitBreakerIgnoreMarker {

    public DuplicateLikeException() {
        super(CommonErrorCode.DUPLICATE_LIKE);
    }
}

File: module-common/src/main/kotlin/maple/expectation/error/exception/DuplicateLikeException.kt
@file:JvmName("DuplicateLikeException")
package maple.expectation.error.exception

import maple.expectation.error.CommonErrorCode
import maple.expectation.error.exception.base.ClientBaseException

class DuplicateLikeException(characterOcid: String) : ClientBaseException(
    CommonErrorCode.DUPLICATE_LIKE,
    characterOcid
)
